engine/src/flutter/lib/ui/painting.dart

```diff
@@ -193,6 +234,13 @@ class Color {
   /// * Bits 16-23 are the red value.
   /// * Bits 8-15 are the green value.
   /// * Bits 0-7 are the blue value.
+  ///
+  /// > [!WARNING]
+  /// > The value returned by this getter implicitly converts normalized
+  /// > component values (such as `0.5`) into their 8-bit equivalent by using
+  /// > the [toARGB32] method; the returned value is not guaranteed to be stable
+  /// > across different platforms or executions due to the complexity of
+  /// > floating-point math.
```

Comment:
Might be worth noting that the returned value is compatible with the default constructor (`Color.new`) but does not guarantee to result in the same color due to imprecisions in numeric conversions.

Reply:
Great suggestion, done (albeit done in toARGB32, this is now a stub method as suggested above)